packages/realm-react/src/__tests__/useQueryHook.test.tsx

```diff
@@ -111,4 +97,74 @@ describe("useQueryHook", () => {
 
     expect(collection[99]).toBe(undefined);
   });
+
+  it("can filter objects via a query argument", () => {
```

Comment:
What do you think of updating the `..via a query option` and `..via a query argument` test names to e.g. `..via an option argument` and `..via separate type and callback arguments`. I initially interperted the "query argument" part as testing for e.g. `$0` etc in the query.

Reply:
I updated the names 🤞